cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -65,7 +65,7 @@ spec:
     # serve the dashboard under a subpath (useful when you are accessing the dashboard via a reverse proxy)
     # urlPrefix: /ceph-dashboard
     # serve the dashboard at the given port.
-    # port: 8443
+    port: 7000
```

Comment:
The default example dashboard specifies `ssl: true`, which means port 8443 should be the default, not 7000.

Reply:
Default example also creates the service which uses `targetPort: 7000`. So I'm not sure how this service should work then if port `8443` is default.

Reply:
As [SSL](https://github.com/rook/rook/blob/ccb4221cb3396d159e94553472777ccfe2f4d925/cluster/examples/kubernetes/ceph/cluster.yaml#L70) is true by default `ssl: true`, the `rook-ceph-mgr-dashboard` service will be created at `8443` port so it's the default port, and if it's set to false `ssl: false` then `rook-ceph-mgr-dashboard` service will be created at or updated to  `7000` port.